web/src/components/netflow-traffic.tsx

```diff
@@ -738,6 +792,7 @@ export const NetflowTraffic: React.FC<{
           reporter,
           setReporter,
           allowReporterBoth: selectedViewId === 'table',
+          useTopK: selectedViewId === 'overview',
```

Comment:
Should we do that as well for topology? (as we discussed in f2f)
what do you think?

Reply:
We could apply a similar approach for topology but not as is.
`top 5` in topology could result only in 3 nodes (or even less with grouping) since each metric represent a single flow direction.